.tours/welcome-to-lightning-web-components-recipes.tour

```diff
@@ -0,0 +1,46 @@
+{
+  "title": "Welcome to Lightning Web Components Recipes",
+  "steps": [
+    {
+      "file": "force-app/main/default/lwc/hello/hello.js",
+      "line": 4,
+      "description": "Hello! \n\nWelcome to Lightning Web Components (LWC) Recipes sample app. \n\nThis demonstrates the use of LWC with a set of code recipes. Each recipe does this with a complete working example, using the minimum amount of code required, while still following best practices. \n\nThis tour will show you around the repo at a high level, and then give you suggestions of what to do next. \n\nIn this file, you can see our most simple \"Hello World\" recipe. "
+    },
+    {
+      "file": "force-app/main/default/lwc/helloBinding/helloBinding.js",
+      "line": 1,
+      "description": "Recipes can be very simple. For instance all the \"Hello\" themed recipes demonstrate basic primitives for how LWC works. \n\nThe core file for any LWC is its JavaScript module, which you can see here for the `helloBinding` LWC. "
+    },
+    {
+      "file": "force-app/main/default/lwc/helloBinding/helloBinding.html",
+      "line": 7,
+      "description": "The template for an LWC is contained in a `.html` file that shares a name with the JavaScript module file. \n\nProperties and functions from the JavaScript are referenced in the template with `{brackets}`, like you see here with `{greeting}` and `{handleChange}`. "
+    },
+    {
+      "file": "force-app/main/default/lwc/helloBinding/helloBinding.js-meta.xml",
+      "line": 2,
+      "description": "All the files of a LWC are contained in a single directory. Together they are called a component bundle. \n\nIn addition to the `.js` and `.html` files, this can also include CSS, submodules, alternate templates, and more. \n\nThis `.xml` file here defines how the LWC is used by the Salesforce platform. \n\nFor more details on the structure of an LWC, check out the LWC developer [guide](https://developer.salesforce.com/docs/component-library/documentation/en/lwc). ",
+      "selection": {
```

Comment:
You are selecting the whole line, is this intentional? I think this is equivalent to just indicate the line number. I think selection is more suitable for when you want to highlight a specific word or similar.

Reply:
yikes. I didn't know I did that. I'll look at this.